### PR TITLE
P0 card component

### PR DESCRIPTION
**Pull request by Naveenchandar**
image

### CODE REVIEW

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

**Review by maitrakhatri — APPROVED** [review]
looks good

File: card.css [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+.basic_card_text_overlap,
+.basic_card_with_header,
+.basic_card_with_btns,
+.basic_card_with_icons
+{
+    background-color: var(--white);
+    border: 1px transparent;
+    border-radius: 7px;
+    width: 21rem;
+}
```
Comment (smilegupta) [inline card.css]: Are these some sort of common properties? if yes they why dont you add all in card property itself

File: card.css [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+.card > div { 
+    margin: 1rem;
+}
+
```
Comment (smilegupta) [inline card.css]: Don't write a generic style...it can hamper in code maintainability

File: card.css [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+.card_container_horizontal{
+    display: flex;
+    align-items: flex-start;
+}
+
+.card_image img,
+.basic_card_image img
+{
```
Comment (smilegupta) [inline card.css]: Dont add properties to the html component only. It can hamper code maintainability

File: card.css [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+    width: 100%;
+    border-top-left-radius: 7px;
+    border-top-right-radius: 7px;
```
Comment (smilegupta) [inline card.css]: Can you a short hand here

File: card.css [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+.card_body h3{
+    margin: 0;
+}
+
+.card_body p{
+    color: var(--gray);
+    margin: 0.5rem 0;
+}
```
Comment (smilegupta) [inline card.css]: same comment as above

File: card.css [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+.basic_card_with_header > .card_header  > h3 {
+    margin: 0;
+}
+.basic_card_with_header > .card_header  > p {
+    margin: 0.5rem 0 0;
+}
+
+.basic_card_with_header > .card_header  > p {
+    color: #00000099;;
+}
```
Comment (smilegupta) [inline card.css]: same comment as above. don't nest styles at all places

File: card.css [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+.card_footer ul .card_more_option{
+    transform: rotate(90deg);
+}
+
+.basic_card_with_btns > .card_footer ul{
+    justify-content: flex-start;
+}
+
+.basic_card_with_icons > .card_footer ul { 
+    justify-content: flex-end;
```
Comment (smilegupta) [inline card.css]: don't nest classes in the entire file..try to use specific classes at componets. Alot of code can be resused here

File: card.css [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+
+.card_footer ul li:nth-child(4){
+    width: 1rem;
+    height: 1rem;
+}
+
+.card_footer ul li:nth-child(5){
+    font-size: 1.25rem;
+}
+
```
Comment (smilegupta) [inline card.css]: Same comment as above. Dont make your css this obvious. Make small typography properties and margin and height spacers where ever required

File: card.css [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+    color: var(--white);
+    font-weight: 600;
+}
+
+.ecommerce_card {
+    display: flex;
+    /* align-items: center; */
+    flex-direction: column;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.2);
+    width: 17rem;
```
Comment (smilegupta) [inline card.css]: remove the unused code

move box shadow in a variable

**Review by smilegupta — COMMENTED** [review]
You can work on the way your write css. You code is functional but its not readbale. Focus on writing clean code alse

File: card.css (hunk 2 above)
Comment (Naveenchandar) [reply card.css]: Can you please explain how it reduces the maintainability.  Idea here is to create a space for the child.

File: card.css (hunk 3 above)
Comment (Naveenchandar) [reply card.css]: Same as above

File: card.css (hunk 9 above)
Comment (Naveenchandar) [reply card.css]: That's not needed. If its used more than once, I can create a variable.

**Pull request merged**